travis_install.sh

```diff
@@ -15,10 +15,8 @@ backend() {
 
 echo "installing $RUNTEST dependencies"
 if [ "$RUNTEST" == "frontend" ]; then
-    frontend
-elif [ "$RUNTEST" == "backend" ]; then
+   	frontend
```

Comment:
Minor, excess indentation here.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/4025/commits/609833ad8f4a91fce45a5ffeab97cfb81cf284c3